packages/autocomplete-plugin-recent-searches/src/createRecentSearchesPlugin.ts

```diff
@@ -50,19 +52,31 @@ export function createRecentSearchesPlugin({
             return store.getAll();
           },
           templates: {
-            item({ item }) {
-              return `
-                <div class="autocomplete-item">
-                  <div>
-                    <svg viewBox="0 0 22 22" width="16" height="16" fill="currentColor">
-                      <path d="M0 0h24v24H0z" fill="none"/>
-                      <path d="M13 3c-4.97 0-9 4.03-9 9H1l3.89 3.89.07.14L9 12H6c0-3.87 3.13-7 7-7s7 3.13 7 7-3.13 7-7 7c-1.93 0-3.68-.79-4.94-2.06l-1.42 1.42C8.27 19.99 10.51 21 13 21c4.97 0 9-4.03 9-9s-4.03-9-9-9zm-1 5v5l4.28 2.54.72-1.21-3.5-2.08V8H12z"/>
-                    </svg>
+            item({ item, root }) {
+              const icon = document.createElement('div');
+              icon.className = 'item-icon aa-RecentSearches--icon';
+              icon.innerHTML = recentIcon;
+              const title = document.createElement('div');
+              title.className = 'aa-RecentSearches--title';
+              title.innerText = item.query;
+              const spacer = document.createElement('div');
```

Comment:
Why do we need a spacer? [`space-between`](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content) on the container should work if `icon` and `title` are in a container.

Reply:
done here https://github.com/algolia/autocomplete.js/pull/326/commits/2acaf862053cd80c0af886402022d02965b29919

FYI, to apply `justify-content: space-between` to the `.aa-Item` level without affecting other sources, I added an extra className `.aa-RecentSearchesItem`.